### ISSUE TITLE
Docs: When a large number of photos are not reconstructed can the process be redone?

### ISSUE THREAD

**Issue by Baasje85** [alicevision/meshroom#491]
I have been batch wise adding images, which I expected a high correlation. 
![image](https://user-images.githubusercontent.com/51262792/58750073-dcbade00-848d-11e9-9385-4ab99dcdb088.png)

But still some photos from the first batch have not been reconstructed. What could be expected from the SfM step? Will it incrementally solve the positions from the earlier groups of images? If not, how could these earlier batches be processed again?

**Comment by fabiencastan** [comment]
> Will it incrementally solve the positions from the earlier groups of images?

Yes, the new images can create new links and could enable to localize cameras that were not solved previously.
